client/src/store/historyStore/model/filtering.js

```diff
@@ -191,6 +191,52 @@ export function getQueryDict(filterText) {
     return queryDict;
 }
 
+/** Returns a dictionary resembling filterSettings (for HistoryFilters):
```

Comment:
This also looks like a duplication, this time of `getFilters`?

Reply:
Right, but it basically parses the final result differently, as explained in the description above the function. The returned `result` variable is different in both functions, as are the `field` and `settingsField` variables.

Reply:
Another PR coming up to fix this.